### PR TITLE
docs: fix contributors list to include all contributors

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/5356019696

Before
image

Now
image

### CODE REVIEW

File: src/storybook/stand-alone-documentaion/welcome/other-contributors-list/other-contributors-list.jsx [hunk 1]
```diff
@@ -47,23 +48,56 @@ const STATIC_CONTRIBUTERS = [
...
+    getAllContributors().then(contributors => setContributorsJson(contributors));
   }, []);
 
   const contributors = useMemo(() => {
     if (contributorsJson && Array.isArray(contributorsJson)) {
       // developer contributors
       const developerContributors = contributorsJson
         .filter(contributor => !excludedDevelopers.has(contributor.id))
-        .map(contributor => ({ name: contributor.login, href: contributor.html_url, key: contributor.id }));
+        .sort((a, b) => b.contributions - a.contributions)
```
Comment (YossiSaadi) [inline src/storybook/stand-alone-documentaion/welcome/other-contributors-list/other-contributors-list.jsx:95]: nice addition 💪🏼

File: src/storybook/stand-alone-documentaion/welcome/other-contributors-list/other-contributors-list.jsx [hunk 2]
```diff
@@ -47,23 +48,56 @@ const STATIC_CONTRIBUTERS = [
...
+  let contributors = [];
+  let page = 1;
+  let list;
+  do {
+    list = await getContributors(page++);
+    contributors = contributors.concat(list);
+  } while (list.length > 0);
+
+  return contributors;
+}
```
Comment (YossiSaadi) [inline src/storybook/stand-alone-documentaion/welcome/other-contributors-list/other-contributors-list.jsx:82]: how long does it take? how many pages there are?

this is something that can be done with `Promise.all` to run the fetches on parallel (collect promises and run them with Promise.all)

If it takes almost no-time, then it's cool

Comment (SergeyRoyt) [reply src/storybook/stand-alone-documentaion/welcome/other-contributors-list/other-contributors-list.jsx:82]: 104 contributors - 2 pages. Seems that it's working almost in no-time

Comment (YossiSaadi) [reply src/storybook/stand-alone-documentaion/welcome/other-contributors-list/other-contributors-list.jsx:82]: so cool for now 💪🏼

**Review by YossiSaadi — APPROVED** [review]

**Pull request merged**